Firmware/mmu2.h

```diff
@@ -297,6 +307,7 @@ class MMU2 {
     ErrorCode lastErrorCode = ErrorCode::MMU_NOT_RESPONDING;
     ErrorSource lastErrorSource = ErrorSource::ErrorSourceNone;
     Buttons lastButton = Buttons::NoButton;
+    uint16_t lastReadRegisterValue = 0;
```

Comment:
Not important now, but may be make a pair<uint8_t register index, uint16_t value> - to make sure the last read value is related to a specific register index. This will become important when we start reading multiple registers on demand.